pkgs/development/python-modules/pywebcopy/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildPythonPackage,
+  pytestCheckHook,
+  setuptools,
+  cachecontrol,
+  lxml-html-clean,
+  requests,
+  six,
+}:
+
+buildPythonPackage rec {
+  pname = "pywebcopy";
+  version = "7.0.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "rajatomar788";
+    repo = "pywebcopy";
+    rev = "v${version}";
+    hash = "sha256-XTPk3doF9dqImsLtTB03YKMWLzQrJpJtjNXe+691rZo=";
+  };
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    cachecontrol
+    lxml-html-clean
+    requests
+    six
+  ];
+
+  meta = {
```

Comment:
I think we should re-add `mainProgram`

Reply:
Why?

Reply:
> meta.mainProgram must be set to the name of the executable which facilitates the primary function or purpose of the package, if there is such an executable in $bin/bin/ (or $out/bin/, if there is no "bin" output). 

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

Reply:
Even if it has not a bin executable but it is only a py module?

Reply:
I havent looked but according to readme there is a CLI

https://github.com/rajatomar788/pywebcopy/?tab=readme-ov-file#command-line-interface

Reply:
ok looks like they don't actually provide a bin script, in which case it's up to you, leave as is, or create a bin script that is essentually `python3 -m pywebcopy $@`

Reply:
I would leave as-is